test/e2e/actions/deploy/deploy_operator.go

```diff
@@ -107,8 +130,117 @@ func MultiNamespaceOperator(data *model.TestDataProvider, watchNamespace []strin
 	})
 }
+
+func DeleteProject(testData *model.TestDataProvider) {
+	By("Delete Project", func() {
+		projectId := testData.Project.Status.ID
+		Expect(testData.K8SClient.Get(testData.Context, types.NamespacedName{Name: testData.Project.Name, Namespace: testData.Project.Namespace}, testData.Project)).Should(Succeed(), "Get project failed")
+		Expect(testData.K8SClient.Delete(testData.Context, testData.Project)).Should(Succeed(), "Delete project failed")
+		aClient := atlas.GetClientOrFail()
+		Eventually(func() bool {
+			return aClient.IsProjectExists(projectId)
+		}).WithTimeout(5*time.Minute).WithPolling(20*time.Second).Should(BeTrue(), "Project was not deleted in Atlas")
+	})
+}
+
+func DeleteUsers(testData *model.TestDataProvider) {
+	By("Delete Users", func() {
+		for _, user := range testData.Users {
+			Expect(testData.K8SClient.Get(testData.Context, types.NamespacedName{Name: user.Name, Namespace: user.Namespace}, user)).Should(Succeed(), "Get user failed")
+			Expect(testData.K8SClient.Delete(testData.Context, user)).Should(Succeed(), "Delete user failed")
+		}
+	})
+}
+
+func DeleteInitialDeployments(testData *model.TestDataProvider) {
+	By("Delete initial deployments", func() {
+		for _, deployment := range testData.InitialDeployments {
+			projectId := testData.Project.Status.ID
+			deploymentName := deployment.Spec.DeploymentSpec.Name
+			Expect(testData.K8SClient.Get(testData.Context, types.NamespacedName{Name: deployment.Name,
+				Namespace: testData.Resources.Namespace}, deployment)).Should(Succeed(), "Get deployment failed")
+			Expect(testData.K8SClient.Delete(testData.Context, deployment)).Should(Succeed(), "Deployment %s was not deleted", deployment.Name)
+			aClient := atlas.GetClientOrFail()
+			Eventually(func() bool {
+				return aClient.IsDeploymentExist(projectId, deploymentName)
+			}).WithTimeout(15*time.Minute).WithPolling(20*time.Second).Should(BeFalse(), "Deployment should be deleted in Atlas")
+		}
+	})
+}
+
+func Project(testData *model.TestDataProvider) {
+	if testData.Project.GetNamespace() == "" {
+		testData.Project.Namespace = testData.Resources.Namespace
+	}
+	By(fmt.Sprintf("Deploy Project %s", testData.Project.GetName()), func() {
+		err := testData.K8SClient.Create(testData.Context, testData.Project)
+		Expect(err).ShouldNot(HaveOccurred(), "Project %s was not created", testData.Project.GetName())
+		Eventually(kube.GetReadyProjectStatus(testData)).WithTimeout(5*time.Minute).WithPolling(20*time.Second).
+			Should(Not(Equal("False")), "Project %s should be ready", testData.Project.GetName())
+	})
+	By(fmt.Sprintf("Wait for Project %s", testData.Project.GetName()), func() {
+		Eventually(func() bool {
+			statuses := kube.GetProjectStatus(testData)
+			return statuses.ID != ""
+		}, 5*time.Minute, 5*time.Second).Should(BeTrue(), "Project %s is not ready", kube.GetProjectStatus(testData))
+	})
+}
+
+func InitialDeployments(testData *model.TestDataProvider) {
```

Comment:
I think this and similar functions should renamed to signal more of an action, like `CreateInitialDeployments` 💭

Reply:
When someone use this function, it will look like deploy.InitialDeployments(testData)

Reply:
Well, ok, still might be a good change but feel free to leave as is for now 👌

Reply:
Done